src/room/track/LocalTrack.ts

```diff
@@ -65,10 +65,23 @@ export default abstract class LocalTrack extends Track {
   }
 
   encryptTrack() {
+    if (!this.sender) {
```

Comment:
Only minor comment: is it possible to do this browser/API detection once and save it rather than every time `encryptTrack` is called? I imagine it's not super expensive, but I wonder if it'd be possible to avoid some of the conditionals and setup steps here.

Or is `encryptTrack` only called once? (I thought it was called every segment?)

Reply:
`encryptTrack` is only called once per "track" basically during initialization (definitely not every segment)